<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoStabilizacjaCzujki</class>
 <widget class="QDialog" name="OknoStabilizacjaCzujki">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Parametry Testu - oczekiwanie na stabilizację czujki</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="head">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Test - oczekiwanie na stabilizację czujki</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="eTestName">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nazwa testu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="testName">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>......</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="podTestName">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lczas">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Czas stabilizacji czujki</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="czas">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>48</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbBreak">
     <property name="text">
      <string>Przerwij</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
